addons/auth_oauth/views/auth_oauth_templates.xml

```diff
@@ -1,35 +1,47 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+        <template id="auth_oauth.assets" inherit_id="web._assets_frontend_helpers" name="OAuth Providers Assets">
```

Comment:
- `web._assets_frontend_helpers` is meant to contain bootstrap variables customization *only*. You want to put your CSS rules in `web.assets_frontend`
- As ID, especially for assets, use the same name you are inheriting from and do not add the name of the app again, it will be done automatically and might only  break stuff if you put the wrong app name by yourself. So here: `id="assets_frontend"`